app/Infrastructure/Http/Resources/BirthdayResource.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Toby\Infrastructure\Http\Resources;
+
+use Illuminate\Http\Resources\Json\JsonResource;
+
+class BirthdayResource extends JsonResource
+{
+    public static $wrap = null;
+
+    public function toArray($request): array
+    {
+        $upcomingBirthday = $this->upcomingBirthday();
+
+        return [
+            "id" => $this->id,
+            "displayDate" => $upcomingBirthday->toDisplayString(),
+            "relativeDate" => $upcomingBirthday->diffForHumans(),
```

Comment:
It would be great if "za 14 godzin" could be transformed into "jutro". In theory this should work: `->diffForHumans(['options' => Carbon::ONE_DAY_WORDS])`, but I'm testing it and I'm not sure it works as intented.

Reply:
We just tested it and it doesn't work

Reply:
I with @Baakoma tested it, but we couldn't transform "za 14 godzin" into "jutro". We can transform "za 1 dzień" into "jutro".

Reply:
And waht about that?
```suggestion
            "relativeDate" => $upcomingBirthday->endOfDay()->diffForHumans(["options" => CarbonInterface::ONE_DAY_WORDS]),
```